backend/messaging-api/messaging-api.yaml

```diff
@@ -0,0 +1,190 @@
+openapi: 3.0.1
+info:
+  title: Feature Messaging API
+  description: This is an OpenAPI document for featurehub messaging data. It is used to publish feature updates to messaging listeners.
+  version: "1.1.1"
+paths:
+  /v2/fake:
+    get:
+      operationId: fakeApi
+      tags:
+        - FeatureMessagingService
+      description: "this is just a holder, it is not generated"
+      responses:
+        200:
+          description: "yay"
+components:
+  schemas:
+    FeatureMessagingUpdate:
+      x-cloudevent-type: messaging-feature-v1
+      x-cloudevent-subject: io.featurehub.events.messaging
+      x-publish-include: messaging
+      x-package: io.featurehub.messaging.model
+      description: "Changes made to a specific feature and who changed it"
+      type: object
+      required:
+        - environmentId
+        - applicationId
+        - whenUpdated
+        - whoUpdated
+        - featureKey
+        - whatUpdated
+      properties:
+        targetDestination:
+          description: "If there is a specific target for the data, everything else _should_ ignore it"
+          type: string
+        environmentId:
+          x-basename: eId
+          type: string
+          format: uuid
+        portfolioId:
+          x-basename: pId
+          type: string
+          format: uuid
+        organizationId:
+          x-basename: oId
+          type: string
+          format: uuid
+        applicationId:
+          x-basename: aId
+          type: string
+          format: uuid
+        whenUpdated:
+          type: string
+          format: date-time
+        whoUpdated:
+          type: string
+          description: Name of the user who made the change to the feature
+        featureKey:
```

Comment:
can we include the featureId and the featureValueId in here as well? The key can change over time, but the two IDs don't.